src/test/java/net/library/controller/ControllerTest.java

```diff
@@ -1,16 +1,21 @@
 package net.library.controller;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 
-import static net.library.util.HttpUtil.BASE_URI;
+import static net.library.util.HttpUtil.GLOBAL_BASE_URI;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
+@ActiveProfiles("test")
+@ExtendWith(SpringExtension.class)
```

Comment:
Why do you need to use it?

Reply:
@ActiveProfiles("test")  or @ExtendWith(SpringExtension.class)?